ci-operator/step-registry/vsphere/deprovision/customized-resourcepool/vsphere-deprovision-customized-resourcepool-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "vsphere/deprovision/customized-resourcepool/vsphere-deprovision-customized-resourcepool-ref.yaml",
```

Comment:
Actually this step is not deprovision resource pool, but run checking, so need to rename the step name.